extensions/3DTILES_bounding_volume_S2/README.md

```diff
@@ -0,0 +1,459 @@
+  ],
+  "root": {
+    "boundingVolume": {
+      "extensions": {
+        "3DTILES_bounding_volume_S2": {
+          "token": "3",
+          "minimumHeight": 0,
+          "maximumHeight": 1000000
+        }
+      }
+    },
+    "refine": "REPLACE",
+    "geometricError": 50000,
+    "children": [
+      {
+        "boundingVolume": {
+          "extensions": {
+            "3DTILES_bounding_volume_S2": {
+              "token": "2c",
+              "minimumHeight": 0,
+              "maximumHeight": 500000
+            }
+          }
+        },
+        "refine": "REPLACE",
+        "geometricError": 500000,
+        "children": [
+          {
+            "boundingVolume": {
+              "extensions": {
+                "3DTILES_bounding_volume_S2": {
+                  "token": "2f",
+                  "minimumHeight": 0,
+                  "maximumHeight": 250000
+                }
+              }
+            },
+            "refine": "REPLACE",
+            "geometricError": 250000,
+            "children": [
+              {
+                "boundingVolume": {
+                  "extensions": {
+                    "3DTILES_bounding_volume_S2": {
+                      "token": "2ec",
+                      "minimumHeight": 0,
+                      "maximumHeight": 125000
+                    }
+                  }
+                },
+                "refine": "REPLACE",
+                "geometricError": 125000
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Implicit Subdivision
+
+When used with [`3DTILES_implicit_tiling`](../3DTILES_implicit_tiling), a `QUADTREE` subdivision scheme will follow the rules for subdivision as defined by the S2 cell hierarchy. When an `OCTREE` subdivision scheme is used, the split in the vertical dimension occurs at the midpoint of the `minimumHeight` and `maximumHeight` of the parent tile.
+
+| Cell  | Quadtree Subdivision | Octree Subdivision |
+|---|---|---|
+| ![Parent Cell](figures/parent.png)  | ![Quadtree Cells](figures/quadtree.png)  | ![Octree Cells](figures/octree.png)  |
+
+To ensure continuity of the Hilbert curve, the faces of the cube are rotated as shown in the diagram below. This must be carefully considered when interpreting the tile coordinates in implicit tiling, since the traversal order in the odd-numbered faces is the mirror of the order in even numbered faces.
+
+```
+                  ^---->  <----^
+                  | 4  |    5  |
+                  |    |       |
+                  x    V  ----->
+
+          ^---->  <----^
+          | 2  |    3  |
+          |    |       |
+          x    V  ----->
+
+  ^---->  <----^
+  | 0  |    1  |
+  |    |       |
+  x    v  ----->
+```
+
+
+### Availability
+
+When using this extension with `3DTILES_implicit_tiling`, the availability bitstreams must be indexed in Morton order, as illustrated by the following diagram:
+
+![Availability](figures/availability.jpg)
```

Comment:
It would be better if this diagram had the fractal curves overlaid for comparison like we did in `3DTILES_implicit_tiling`. I have a script for the Morton curve, but I'd have to adapt it for Hilbert curves too.